common/src/main/java/net/dodogang/plume/Plume.java

```diff
@@ -19,15 +20,16 @@
 import net.minecraft.item.ItemStack;
 import net.minecraft.item.Items;
 import net.minecraft.world.poi.PointOfInterestType;
+
 import org.apache.logging.log4j.Level;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
```

Comment:
I'm not a huge fan of separating imports like this, but I can't find any way to combine different prefix groups in clang-format

Reply:
I don't mind either of the styles, I never really see imports as I just have them minimised :)

Reply:
Same